cms/envs/aws.py

```diff
@@ -312,3 +312,7 @@
 #date format the api will be formatting the datetime values
 API_DATE_FORMAT = '%Y-%m-%d'
 API_DATE_FORMAT = ENV_TOKENS.get('API_DATE_FORMAT', API_DATE_FORMAT)
+
+if FEATURES['ENABLE_COURSEWARE_INDEX']:
+    # Use ElasticSearch for the search engine
+    SEARCH_ENGINE = "search.elastic.ElasticSearchEngine"
```

Comment:
If SEARCH_ENGINE is undefined or None, then indexing is skipped. This allows tests to run as fast as possible when they do not depend upon adding information to the index.